ui/StatusQ/src/StatusQ/Controls/StatusTabButton.qml

```diff
@@ -30,14 +30,16 @@ TabButton {
     leftPadding: 12
     rightPadding: 12
 
-    background: Item { }
+    background: MouseArea {
```

Comment:
You could just add a `HoverHandler` under this background/Item; we have way too many MouseAreas everywhere :)

Reply:
Great tip! TIL that HoverHandler has cursorShape property ;)

Reply:
That's its main usage imo :)

Reply:
`cursorShape` property was introduced to HoverHandler in 5.15, so I always encountered MouseAreas for cursor handling :)

Reply:
Ah yes, that's right